front/components/modal/excludedNodes.tsx

```diff
@@ -87,16 +87,6 @@ const ExcludedNodes: ExcludedNodesComponent = ({
     })
   }
 
-  // TODO : Improve error management
-  useEffect(() => {
```

Comment:
Du coup where did it go ?

Reply:
J'ai enlevé parce qu'il y avait la notif + le message d'erreur